springwolf-examples/springwolf-cloud-stream-example/src/main/java/io/github/springwolf/examples/cloudstream/configuration/CloudstreamConfiguration.java

```diff
@@ -31,6 +33,7 @@ public Consumer<AnotherPayloadDto> consumerMethod() {
 
     @GooglePubSubAsyncChannelBinding(
             schemaSettings = @GooglePubSubAsyncSchemaSetting(encoding = "BINARY", name = "project/test"))
+    @GooglePubSubAsyncMessageBinding(schema = @GooglePubSubAsyncMessageSchema(name = "project/test"))
```

Comment:
Great to update the example. This should update the AsyncAPI.json as well. Is the dependency in the examples build gradle file missing?

Reply:
Or rather the class is missing in the bindings autoconfiguration

Reply:
sure I will update example and yes binding is missing from auto-configuration